tests/test_input.py

```diff
@@ -0,0 +1,26 @@
+"""
+"""
+
+import unittest
+import networkx as nx
+from probinet.input.loader import build_adjacency_from_networkx
+from probinet.models.classes import GraphData
+
+
+class TestInput(unittest.TestCase):
+
```

Comment:
Somewhere here we should add a `setUp` method that creates a temporary folder, and then pass it to the function we are testing. This folder disappears once the test is finished. That way, we won't be left with some auxiliary files every time the test is run.